jubakit/loader/postgresql.py

```diff
@@ -0,0 +1,95 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from ..base import BaseLoader
+from ..compat import *
+from psycopg2 import connect
+from psycopg2.extras import DictCursor
+from psycopg2 import sql
+
+class PostgreSQLoader(BaseLoader):
+  """
+  Loader to process columns of PostgreSQL.
+
+  This loader that load data from PostgreSQL's table as below.
+  We access the "test" table of the "test" database in the below example.
+
+  Example:
+    from jubakit.loader.postgresql import PostgreSQLoader
+    from jubakit.loader.postgresql import PostgreSQLAuthHandler
+    
+    auth = PostgreSQLAuthHandler(dbname='test', user='postgres', password='postgres', host='localhost', port='5432')
+    
+    loader = PostgreSQLoader(auth, table='test')
+    for row in loader:
+      print(row)
+    
+    # {1: 'id', 100: 'num', 'abcdef': 'data'}
```

Comment:
I think the key-value pairs are reversed.
Is `{'id': 1, 'num': 100, 'data': 'abcdef'}` correct?